### DISCUSSION TITLE
AG File Browser

### DISCUSSION THREAD

**Discussion by abysalvinthomas** [apohl79/audiogridder#665] [Ideas]
Hi
1.2 beta 4 is works excellent. Great update.
Hear i have one suggestion for upcoming update,
a file browser plugin or an attached window.
i think its so useful..because. in my case i have kept all my loops (wave files) in my server HDD. now i am struggling to call that file to my DAW. and some of the Instruments like KONTAKT , BATTERY & SLISEX need drag and drop for easy workflow.
at the same time ..If an option to drag and drop files direct to DAW will be fine.
i think ..i conveyed my idea..
expecting a positive note
thank you.

**Comment by apohl79** [comment]
I don‘t get what you are asking for tbh. If you want to drag&drop into Kontakt for example running remotely, you can extend the screen capture area of the plugin and than drag a file into the Kontakt window. Not sure what AG should do. Also I don‘t understand the DAW part.

**Reply by abysalvinthomas** [reply]
a window in AG plugin for Drag files (wave) to DAW (from server side to master DAW)

**Reply by apohl79** [reply]
You can do this via screen capturing as described before. Increase the captured area beyond the plugin window, open a file browser and drag the files into the plugin on the server side.

**Reply by abysalvinthomas** [reply]
let me clear.....my idea..is to drag a file from server side to master DAW (to the track of Master DAW)

**Reply by apohl79** [reply]
OK, thats nothing AG can support, as it is a network bridge for plugins and does not have access to audio files the DAW works with. You can mount a network drive to your DAW computer to access remote audio files on your server from your DAW.

**Comment by abysalvinthomas** [comment]
ok
Thank you dear.
